data/logs_model/splunk_logs_model.py

```diff
@@ -0,0 +1,85 @@
+import json
+import logging
+
+from datetime import datetime
+
+from data import model
+from data.logs_model.logs_producer import LogProducerProxy, LogSendException
+from data.logs_model.logs_producer.splunk_logs_producer import SplunkLogsProducer
+from data.logs_model.shared import SharedModel
+
+logger = logging.getLogger(__name__)
+
+
+class SplunkLogsModel(SharedModel):
```

Comment:
I think you'll need to implement at least _some_ of the methods in `ActionLogsDataInterface`. I found that `lookup_logs` for example is called by `endpoints/api/superuser.py` and `endpoints/api/logs.py`. `workers/exportactionlogsworker.py` calls `yield_logs_for_export`. 

These are only the ones I checked - I think you need to grep around the code and see which functions from `ActionLogsDataInterface` are called and which ones aren't. Then you'll need to make a choice on whether the splunk logs model needs them or not (and what happens if you don't write them?). I would assume that the superuser and logs apis need to function when customers use splunk logs, but I don't know for sure. Same question applies to other parts of quay.

Reply:
except for `log_action` every other method in `ActionLogsDataInterface` does a look up of logs from splunk. I confirmed with PM that for this release we only support forwarding logs to splunk. I have added a generic message for the unimplemented methods

Reply:
What does it look like from the perspective of a user of the api? I think raising a `NotImplementedError` will blow up a 500 unless you treat that exception somehow.

Reply:
What features won't be available to customers if they use Splunk?